Odata-docs/client/v6/code-generation-tool.md

```diff
@@ -27,6 +27,7 @@ OData provides three tools to generate client proxy file for an OData Service.
 - [OData Connected Service](https://visualstudiogallery.msdn.microsoft.com/b343d0eb-6493-44c2-b558-13a0408d013f) lets app developers connect their applications to OData Services (both V3 & V4) and generate the client proxy files for the services. It supports following Visual Studio:
 
 	- Visual Studio 2015
```

Comment:
OData Connected Service support VS 2017 and 2019, not 2015. Could you update that too?